tests/test_numberings.py

```diff
@@ -22,6 +22,13 @@ def test_do_not_restart_numbering_of_bullets(mixed_numberings):
     assert len(xpath(paragraphs[10], './/w:numId')) == 0
 
 
+def test_do_not_fail_when_numering_zero_is_referenced():
```

Comment:
typo: `numering` -> `numbering`

There's no assertion and thus it's not so obvious what this test does.
I would do something like this:
```
try:
    composer.append(Document(docx_path("numberings_styles.docx")))
except Exception:
    pytest.fail('Unexpected error...')
```